src/settings/Settings.tsx

```diff
@@ -230,6 +230,20 @@ export const SETTINGS: { [setting: string]: ISetting } = {
             requiresRefresh: true,
         },
     },
+    "feature_notifications_panel": {
+        isFeature: true,
+        labsGroup: LabGroup.Messaging,
+        displayName: _td("Notifications panel"),
+        supportedLevels: LEVELS_FEATURE,
+        default: false,
+        // Reload to ensure that the left panel etc. get remounted
+        controller: new ReloadOnChangeController(),
+        betaInfo: {
```

Comment:
Marking this as a beta implies its nearly ready for prime time, by taking this feature out of mainline stability we're saying it isn't. This will be confusing IMO

Reply:
This is the only way to make it available to people who do not use develop/nightly.
I don't think this will be more confusing than the way it works today